extensions/screen-saver/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "screen-saver",
+  "title": "Screen Saver",
+  "description": "Activate screen saver of your Mac.",
+  "icon": "screen-saver.png",
+  "author": "koinzhang",
+  "categories": [
+    "Productivity",
+    "System"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "screen-saver",
+      "title": "Screen Saver",
+      "subtitle": "System",
```

Comment:
For the sake of this don't getting mixed up with our native System commands, could you rename the command to "Start Screen Saver" and remove the "System" subtitle?

Reply:
Hi @danielsequeira. I have made a few changes.
The **Screen Saver** command was changed to **Start Screen Saver**
The subtitle of the command is changed to **Screen Saver**